src/main/ts/Part2Ex4FP.ts

```diff
@@ -93,6 +93,11 @@ const nextMode = (m: Mode): void => {
   }
 };
 
+export const nextMode1 = (m: Mode): Mode => {
+  if (m === 'code') return 'design'
+  if (m === 'design') return 'markdown'
+  return 'code'
```

Comment:
Thought I would just note that this is closer to how we would write it normally.
```suggestion
  if (m === 'code') {
    return 'design'
  } else if (m === 'design') {
    return 'markdown'
  } else  {
    return 'code'
  }
```

Reply:
Hmm right 🤔
I don't like to write `else if` or `else` when there's a `return` but I can change to follow your code style 👍

Reply:
Yeah, we always make sure an `if` uses brackets (I believe we have a lint rule). With regards to having an `else`, it sort of goes back to functional programming where an `if` must have an `else`. We try to do that as much as possible. Others can correct me if I am wrong.